homeassistant/components/media_player/mpd.py

```diff
@@ -225,11 +248,15 @@ def media_previous_track(self):
 
     def play_media(self, media_type, media_id, **kwargs):
         """Send the media player the command for playing a playlist."""
-        _LOGGER.info(str.format("Playing playlist: {0}", media_id))
-        if media_type == MEDIA_TYPE_PLAYLIST:
```

Comment:
Ok I changed it back. I had changed it in the first place because at least with a mopidy server, if you try to play a playlist that isn't in "the list" it doesn't do well and actually goes into a rapid connect loop until mopidy bans it because too many concurrent connections. That's why I left this warning. I will try to fix that separate issue in a later PR.